lib/exec.js

```diff
@@ -232,11 +232,12 @@ const manifestMissing = (tree, mani) => {
 
 const getBinFromManifest = mani => {
   // if we have a bin matching (unscoped portion of) packagename, use that
-  // otherwise if there's 1 bin, use that,
+  // otherwise if there's 1 bin or all bin value is the same (alias), use that,
   // otherwise fail
-  const bins = Object.entries(mani.bin || {})
-  if (bins.length === 1) {
-    return bins[0][0]
+  const maniBin = mani.bin || {}
+  if (Array.from(new Set(Object.values(maniBin))).length === 1) {
+    const binNameList = Object.keys(maniBin)
+    return binNameList[0]
   }
```

Comment:
Side note: don't know why test coverage drop from 100% with previous code style:
```js
  const bins = Array.from(new Set(Object.values(mani.bin || {})))
  if (bins.length === 1) {
    return Object.keys(mani.bin || {})[0] // test coverage will report this line uncovered
  }
```
But reducing code complexity by extracting variable seems to help here.

Reply:
The reason you get an uncovered branch on that line is because you can't possibly get there if `mani.bin` is missing, so the `mani.bin || {}` will never have a falsey `mani.bin` and thus never hit the `|| {}` branch.

Try running `npm test -- --coverage-report=html` to open up a web browser and see missing branches, just trying to go off of line numbers can be tricky for us human brains ;)